etc/kayobe/ansible/fix-networking.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Fix networking
-  hosts: seed,compute,controllers,storage
+  hosts: "{{ fix_networking_playbook_hosts | default('seed,compute,controllers,storage') }}"
```

Comment:
ditto

Reply:
I've reverted the change and just added the group